eng/pipelines/templates/jobs/ci.yml

```diff
@@ -78,11 +78,17 @@ jobs:
           echo "##vso[build.addbuildtag]Scheduled"
         displayName: "Tag scheduled builds"
         condition: and(eq(variables['Build.SourceBranchName'],variables['DefaultBranch']),eq(variables['Build.Reason'],'Schedule'))
-      # Both 'Dump property properties' and 'Update package properties with dev version' serve the same purpose of generating package json.
-      # We decided to only run the later one (update properties) in all pipelines for the following reasons:
-      # 1. 'Update properties' happens after 'build', we will have more information at the 'update properties' step.
-      # 2. We did not find any usage of package json before 'update properties' in .NET, so it is safe to skip the 'dump properties' for now.
+      # Set env variable SkipDevBuildNumber to generate package properties with non alpha version
+      # Package version is used to create or update package work item from CI pipeline
+      - pwsh: |
+          echo "##vso[task.setvariable variable=SkipDevBuildNumber]true"
+        displayName: "Set SkipDevBuildNumber env variable"
       - template: /eng/common/pipelines/templates/steps/daily-dev-build-variable.yml
+        parameters:
+          ServiceDirectory ${{parameters.ServiceDirectory}}
+      - pwsh: |
+          echo "##vso[task.setvariable variable=SkipDevBuildNumber]false"
```

Comment:
You can put this in the pwsh step right below this. I would also not set it to false but instead set it to empty.